tests/integration/src/install_tests.rs

```diff
@@ -0,0 +1,1009 @@
+        }],
+        name: "station-admin".to_string(),
+        groups: Some(vec![ADMIN_GROUP_ID.hyphenated().to_string()]),
+        id: None,
+        status: None,
+    }],
+    accounts: vec![],
+    assets: vec![],
+    permissions: vec![],
+    request_policies: vec![],
+    user_groups: vec![], // no user groups yet user is referencing the ADMIN group
+    named_rules: vec![],
+    disaster_recovery_committee: None,
+})]
+#[should_panic]
+#[case::circular_named_rules({
+    let id_1 = Uuid::new_v4().hyphenated().to_string();
+    let id_2 = Uuid::new_v4().hyphenated().to_string();
+
+    station_api::InitialConfig::Complete {
+        users: vec![UserInitInput {
+            identities: vec![UserIdentityInput {
+                identity: WALLET_ADMIN_USER,
+            }],
+            name: "station-admin".to_string(),
+            groups: Some(vec![ADMIN_GROUP_ID.hyphenated().to_string()]),
+            id: None,
+            status: None,
+        }],
+        accounts: vec![],
+        assets: vec![],
+        permissions: vec![],
+        request_policies: vec![],
+        user_groups: vec![
+            InitUserGroupInput {
+                id: Some(ADMIN_GROUP_ID.hyphenated().to_string()),
+                name: "admin".to_string(),
+            },
+        ],
+        named_rules: vec![
+            // circular reference
+            InitNamedRuleInput {
+                id: Some(id_1.clone()),
+                name: "named_rule".to_string(),
+                description: None,
+                rule: RequestPolicyRuleDTO::NamedRule(id_2.clone()),
+            },
+            InitNamedRuleInput {
+                id: Some(id_2.clone()),
+                name: "named_rule_2".to_string(),
+                description: None,
+                rule: RequestPolicyRuleDTO::NamedRule(id_1.clone()),
+            },
+        ],
+        disaster_recovery_committee: Some(DisasterRecoveryCommitteeDTO {
+            quorum: 1,
+            user_group_id: ADMIN_GROUP_ID.hyphenated().to_string(),
+        }),
+    }
+})]
+#[should_panic]
+#[case::non_existent_asset_id({
+    let id_1 = Uuid::new_v4().hyphenated().to_string();
+    station_api::InitialConfig::WithDefaultPolicies {
+        users: vec![UserInitInput {
+            identities: vec![UserIdentityInput {
+                identity: WALLET_ADMIN_USER,
+            }],
+            name: "station-admin".to_string(),
+            groups: Some(vec![ADMIN_GROUP_ID.hyphenated().to_string()]),
+            id: None,
+            status: None,
+        }],
+        admin_quorum: 1,
+        operator_quorum: 1,
+        accounts: vec![InitAccountInput {
+            name: "account".to_string(),
+            metadata: vec![],
+            assets: vec![
+                id_1.clone(), // non-existent asset id
+            ],
+            id: Some(id_1.clone()),
+            seed: Uuid::new_v4().to_bytes_le(),
+        }],
+        assets: vec![],
+    }
+})]
+#[should_panic]
```

Comment:
Could we make sure that the panic has the expected cause? (E.g., that the panic is not cause by insufficient cycles)

Reply:
Adding panic reasons